src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -194,6 +194,8 @@ def __init__(self, experiment_name: str, device: str = None, multi_gpu: Union[Mu
         self.max_train_batches = None
         self.max_valid_batches = None
 
+        self.epoch_start_lr_dict = {}
```

Comment:
seem to me that this is only for internal use, please change attribute name to _epoch_start_lr_dict.